scripts/common/object_store.sh

```diff
@@ -257,3 +257,146 @@ EOF
     printf "\n${GREEN}Object store migration completed successfully${NC}\n"
     report_addon_success "rook-ceph-to-minio" "v1"
 }
+
+function migrate_minio_to_rgw() {
```

Comment:
This is an almost equal copy of `migrate_rgw_to_minio` above.

Reply:
I hate to ask you to rewrite this
would it be possible to make both of these into wrapper scripts that set things like "source/dest ns" "source/dest service" "source/dest access key ID" and then call the same underlying logic?